core-src/annotatemain.cc

```diff
@@ -702,7 +715,34 @@ int main(int argc, char **argv)
 
     GameState initialState;
 
-    if (auto_setup && arg_index == argc) {
+    if (load_setup) {
+        struct stat buf;
+        if (stat(file_name, &buf) != 0) {
+            do_error("The loading file doesn't exist.");
+        }
+        g_Verbose = true;
+        FILE * file = fopen(file_name, "r");
```

Comment:
FYI, here you should be checking the return value of `fopen` against `NULL`, and issuing an error if it's not openable and readable. This is strictly more accurate than stat'ing the file, for at least two reasons:

- no TOCTTOU race condition
- correct handling if `file_name` is the name of a directory or special device, e.g. `--load ..`

and preferable for one more reason, namely that it doesn't require pulling in `<sys/stat.h>`.